data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,560 @@
+	to offer
+		not "hr: sandsnap project"
+	source
+		government "Hai"
+		not planet "Makerplace"
+	destination "Makerplace"
+	to complete
+		has "hr: sandsnap project"
+	on complete
+		payment 1000 150
+
+
+
+mission "Hai Reveal Concern Period: Reactivating Sandsnap"
+	priority
+	name "Transport the Stationkeeper to Sandsnap."
+	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
+	source "Makerplace"
+	destination "Sandsnap"
+	passengers 1
+	to offer
+		has "hr: allhome raided"
+	on offer
+		conversation
+			`	Upon entering the spaceport you are immediately greeted by a worried hai, "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren north were right in assuming evil intents."`
+			`	He stops himself, "I must be boring you with politics, let us cut to the matter at hand, shall we?"`
+			choice
+				`	(Nod.)`
+					goto border
+				`	"Not at all, I love politics."`
+			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, captain <last>, the Hai will wake up.`
+			label border
+			`	"The raids have proven that we are not prepared for an offensive on our borders. In order to address this, we want to repair the abandoned station of <destination>. We have already hired a few mercenaries to help with the defense of our station, most of them posing as human merchants. Once operational, the station should be a major strategic advantage for us, allowing fast resupply and repairs of ships." He stops in front of a very distinguished house, "This is the residence of our esteemed Stationkeeper. You will need to escort him to the station so he may make the necessary adjustments and assess exactly what kind of supplies are required. I'll leave you in her company," he says, heading of at a quick pace.`
+			choice
+				`	(Greet the Stationkeeper.)`
+					goto greet
+				`	(Bow before the Stationkeeper.`
+			`	She seems flattered, "There is no need, Captain, my station is not operational, and the same goes for the honor that was due to me. For now, anyway."`
+			label greet
+			`	You introduce yourself, "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks. She does not seem to be the talkative kind.`
```

Comment:
```suggestion
			`	You introduce yourself. "You may refer to me simply as the Stationkeeper. Now, if you don't mind, I have been waiting for this opportunity all my life, so I would really like to get going!" You show her to her bunks and try to make conversation, but unlike the Hai that greeted you earlier, she does not seem to be the talkative kind.`
```
It may be worthwhile to name the Stationkeeper, at least to bridge this bit instead of leaving a gap between your introduction and her introduction.

Why has she been waiting for this opportunity all her life? Is she one of those Hai that has long term goals and not long-term whims? And a Hai life is meant to be quite long...

Also to put this into perspective, you've been accosted by a Hai you don't know and told to deliver another Hai to a deactivated station, and it seems without any payment or way of refusal. (I know it's a draft, but I'd would rather spell it out even for my own sake so if I choose to work on it I'll have some notes to work from.)

Reply:
I think that she can just go by stationkeeper?
But yes the details could use some ironing 100%